docs/resources/google_compute_zone.md

```diff
@@ -1,57 +1,60 @@
+      changes will be clobbered when the file is regenerated.
 
-    describe google_compute_zone(project: 'chef-inspec-gcp',  zone: 'us-east1-b') do
-      it { should exist }
-    end
+      Please read more about how to change this file in README.md and
+      CONTRIBUTING.md located at the root of this package.
 
-### Test that a GCP compute zone is in the expected state
+  ----------------------------------------------------------------------------
+-->
 
-    describe google_compute_zone(project: 'chef-inspec-gcp',  zone: 'us-east1-b') do
-      its('status') { should eq 'UP' }
-      # or equivalently
-      it { should be_up }
-    end
-
-### Test that a GCP compute zone has an expected CPU platform
+---
+title: About the Zone resource
+platform: gcp
+---
 
-    describe google_compute_zone(project: 'chef-inspec-gcp',  zone: 'us-east1-b') do
-      its('available_cpu_platforms') { should include "Intel Skylake" }
-    end
+## Syntax
+A `google_compute_zone` is used to test a Google Zone resource
 
-<br>
+TODO: Examples
 
 ## Properties
-
-*  `available_cpu_platforms`, `creation_timestamp`, `description`, `id`, `kind`, `name`, `region`, `status`, `region_name`
-
-<br>
-
-
-## GCP Permissions
-
-Ensure the [Compute Engine API](https://console.cloud.google.com/apis/library/compute.googleapis.com/) is enabled for the project where the resource is located.
\ No newline at end of file
+Properties that can be accessed from the `google_compute_zone` resource:
+  * `creation_timestamp`: Creation timestamp in RFC3339 text format.
+  * `deprecated`: The deprecation status associated with this machine type.
+    * `deleted`: An optional RFC3339 timestamp on or after which the state of this
+resource is intended to change to DELETED. This is only
+informational and the status will not change unless the client
+explicitly changes it.
+
+    * `deprecated`: An optional RFC3339 timestamp on or after which the state of this
+resource is intended to change to DEPRECATED. This is only
+informational and the status will not change unless the client
+explicitly changes it.
+
+    * `obsolete`: An optional RFC3339 timestamp on or after which the state of this
+resource is intended to change to OBSOLETE. This is only
+informational and the status will not change unless the client
+explicitly changes it.
+
+    * `replacement`: The URL of the suggested replacement for a deprecated resource.
+The suggested replacement resource must be the same kind of
+resource as the deprecated resource.
+
+    * `state`: The deprecation state of this resource. This can be DEPRECATED,
+OBSOLETE, or DELETED. Operations which create a new resource
+using a DEPRECATED resource will return successfully, but with a
+warning indicating the deprecated resource and recommending its
+replacement. Operations which use OBSOLETE or DELETED resources
+will be rejected and result in an error.
+  * `description`: An optional textual description of the resource.
```

Comment:
There should probably be newlines on either side of these properties

(You'd be surprised how many people read the raw Markdown. The compiled Markdown looks nice!)